### PR TITLE
[4.0] fixes inconsistent logo sizes

### PR DESCRIPTION
**Pull request by sksuryan**
Pull Request for Issue #31936.

### Summary of Changes
This pull request fixes the inconsistent Joomla logo sizes in the headers of installation page and administrator page.

### Actual result BEFORE applying this Pull Request
image
image

### Expected result AFTER applying this Pull Request
image
image

### CODE REVIEW

File: installation/template/index.php [hunk 1]
```diff
@@ -65,9 +65,19 @@
 		<div class="j-install">
 			<?php // Header ?>
 			<header id="header" class="header">
-				<div class="d-flex ">
+				<div class="d-flex align-items-center">
 					<div class="logo d-none d-md-block col">
-						<img src="<?php echo $this->baseurl; ?>/template/images/Joomla-logo-monochrome-horizontal-white.svg" alt="">
+						<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 376.3 74.8" x="0" y="0" width="114px" height="20px" transform="translate(10 0) scale(1.3)" role="presentation" focusable="false">
```
Comment (C-Lodder) [inline installation/template/index.php]: How come you're using `transform="translate(10 0) scale(1.3)"`?

**Comment by drmenzelit** [comment]
Thank you for your first PR in Joomla @sksuryan. @C-Lodder  can you be so kind and explain our new contributor (GSoC student) what you mean with your comment? Thank you.

File: installation/template/index.php (hunk 1 above)
Comment (sksuryan) [reply installation/template/index.php]: Hello @C-Lodder, this is the actual code of SVG that is rendered on administrator page (brand-large.svg  in administrator/templates/atum/images/logos) and isn't something I have written myself.
Previously, I tried to use that same SVG itself by copying it to images in installation, but for some reason it was cropping out parts of the logo, and giving margin or padding didn't help either.

**Comment by sksuryan** [comment]
> Thank you for your first PR in Joomla @sksuryan. @C-Lodder  can you be so kind and explain our new contributor (GSoC student) what you mean with your comment? Thank you.

Thank you @drmenzelit, Glad to make my first PR to Joomla.

**Comment by drmenzelit** [comment]
I can confirm that the svg logo has a `transform="translate(10 0) scale(1.3)"` defined... why? I have no idea...
But in the index.php from atum the logo is being inserted in an image tag and not as svg direct. Which way is better it is difficult to say, but we should be consistent.

**Comment by sksuryan** [comment]
image

for some reason, only this logo was being cut-off from all sides like this while rest images were fine but it is fine if I use it's source directly.

I realized that was because I was using img tag but since it's an svg I can directly import it and use it.

> 
> 
> I can confirm that the svg logo has a `transform="translate(10 0) scale(1.3)"` defined... why? I have no idea...
> But in the index.php from atum the logo is being inserted in an image tag and not as svg direct. Which way is better it is difficult to say, but we should be consistent.


Pushing new changes which should address this. It also changes px values in favor of rem.

**Comment by srishty-07** [comment]
I have tested this item :white_check_mark: successfully on 05435747212d6b0f320f71f573d19717f2ee974b<br /><br />I have tested it successfully on Joomla 4.0.0-beta-8-dev.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32632">issues.joomla.org/tracker/joomla-cms/32632</a>.</sub>

File: installation/template/index.php [hunk 2]
```diff
@@ -28,6 +28,8 @@
 	->useScript('webcomponent.joomla-alert')
 	->useScript('webcomponent.core-loader');
 
+$logo = file_get_contents(__DIR__.'/images/brand-large.svg');
+
```
Comment (Quy) [inline installation/template/index.php:32]:
```suggestion
$logo = file_get_contents(__DIR__.'/images/brand-large.svg');
```

File: installation/template/index.php [hunk 3]
```diff
@@ -28,6 +28,8 @@
...
 
 // Add script options
```
Comment (Quy) [inline installation/template/index.php]:
```suggestion
$logo = file_get_contents(__DIR__.'/images/brand-large.svg');

// Add script options
```

File: installation/template/index.php [hunk 4]
```diff
@@ -65,9 +67,9 @@
 		<div class="j-install">
 			<?php // Header ?>
 			<header id="header" class="header">
-				<div class="d-flex ">
+				<div class="d-flex align-items-center">
 					<div class="logo d-none d-md-block col">
-						<img src="<?php echo $this->baseurl; ?>/template/images/Joomla-logo-monochrome-horizontal-white.svg" alt="">
+						<?php echo $logo ?>
```
Comment (Quy) [inline installation/template/index.php]:
```suggestion
						<?php echo $logo; ?>
```

Comment (sksuryan) [reply installation/template/index.php]: requested changes have been made.

**Comment by Quy** [comment]
An alternative PR #33100.

**Comment by sksuryan** [comment]
> 
> 
> An alternative PR #33100.

This PR actually adds the same logo to Installation UI as in the Admin UI since the previous logo in the Installation UI was much bigger. This PR doesn't change anything in the Admin UI itself.

**Comment by richard67** [comment]
@sksuryan Do you really think someone will ever compare the logo sizes of the installation and the admin UI? To be honest, in my personal opinion this PR here is not really needed. But that's just my personal opinion, I don't have any right to decide ;-)

**Comment by sksuryan** [comment]
@richard67 I mean, an issue does exist for this

**Comment by brianteeman** [comment]
@richard67 consistency in branding is always good

**Comment by richard67** [comment]
@sksuryan Not really a major issue, and sometimes we close issues as won't fix. But as said, that's not only my decision.

**Comment by richard67** [comment]
> consistency in branding is always good

@brianteeman Is the same brand, only different size, so not really inconsistent. And the PR adds another import of an SVG .. is that really worth the little benefit?

**Comment by Quy** [comment]
This has been fixed with PR #33100. Please install the latest version and confirm.

**Comment by brianteeman** [comment]
I am making no comment on how the code is changed ;)

If it was just an issue then closing as "wont fix" is fine and should be done much much much more often
But this is a PR.

If the PR works and meets standards etc then it should be considered.

if it has already been fixed elsewhere then of course it should be closed as there is nothing left to fix?

**Comment by alikon** [comment]
that issue is still present after merge of  #33100  ?
anyway as per  Conflicting files @sksuryan you should revisit your pr ;)

**Comment by sksuryan** [comment]
> 
> 
> that issue is still present after merge of #33100 ?
> anyway as per Conflicting files @sksuryan you should revisit your pr ;)

I'll take a look @alikon

**Comment by Quy** [comment]
Closing as this is no longer an issue. Thanks for the PR.

**Pull request closed without merging**